web-registry/src/pages/CreditClassDetails/tables/CreditClassDetails.TableTabs.tsx

```diff
@@ -0,0 +1,11 @@
+import { TableTabs } from 'components/organisms/TableTabs';
+
+import { getCreditClassDetailsTabs } from './CreditClassDetails.TableTabs.config';
+import { CreditClassDetailsTableTabsProps } from './CreditClassDetails.TableTabs.types';
+
+export const CreditClassDetailsTableTabs = (
+  props: CreditClassDetailsTableTabsProps,
+) => {
+  const tabs = getCreditClassDetailsTabs(props);
+  return tabs.length > 0 ? <TableTabs tabs={tabs} /> : null;
```

Comment:
we can make this length check part of the `TableTabs` component itself so we don't have to repeat this logic in both CreditClassDetails and ProjectDetails

Reply:
Updated https://github.com/regen-network/regen-web/pull/1985/commits/6ba6551b524078a06d375862e379cada36586c4e